resources/wikia/modules/krux.js

```diff
@@ -71,13 +71,14 @@ define('wikia.krux', ['wikia.window', 'wikia.document'], function (win, doc) {
 	}
 
 	function getSegments() {
-		var segments = getParams('segs');
+		var segments = getParams('segs'),
+			segsArray = segments.split(',');
 
-		if (segments) {
-			return segments.split(',').slice(0, maxNumberOfKruxSegments);
+		if (segsArray.indexOf('ph3uhzc41') > maxNumberOfKruxSegments - 1) {
```

Comment:
What's `ph3uhzc41`? A comment would be nice to know it in future :japanese_ogre: